apps/mymedicare_cb/authorization.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import requests
+from django.conf import settings
+
+logger = logging.getLogger('hhs_server.%s' % __name__)
+
+
+class OAuth2Config():
+    token_endpoint = settings.SLS_TOKEN_ENDPOINT
+    redirect_uri = settings.MEDICARE_REDIRECT_URI
+    verify_ssl = getattr(settings, 'SLS_VERIFY_SSL', False)
+    token = None
+
+    @property
+    def client_id(self):
+        return getattr(settings, 'SLS_CLIENT_ID', False)
+
+    @property
+    def client_secret(self):
+        return getattr(settings, 'SLS_CLIENT_SECRET', False)
+
+    def basic_auth(self):
+        if self.client_id and self.client_secret:
+            return (self.client_id, self.client_secret)
+        return None
+
+    def exchange(self, code):
+        logger.debug("token_endpoint %s" % (self.token_endpoint))
+        logger.debug("redirect_uri %s" % (self.redirect_uri))
+
+        token_dict = {
+            "grant_type": "authorization_code",
+            "code": code,
+            "redirect_uri": self.redirect_uri,
+        }
+
+        r = requests.post(self.token_endpoint,
```

Comment:
Can we use response in place of r? I know they use r in their documentation, but it can get very confusing.